app/routes/administrative-units/administrative-unit/core-data/edit.js

```diff
@@ -10,6 +10,15 @@ import worshipAdministrativeUnitValidations, {
 
 export default class AdministrativeUnitsAdministrativeUnitCoreDataEditRoute extends Route {
   @service store;
+  @service currentSession;
+  @service session;
+  @service router;
+
+  beforeModel() {
+    if (!this.currentSession.canEdit) {
+      this.router.transitionTo('route-not-found', { wildcard: 'not-found' });
```

Comment:
I think we went with dutch wording for the path names. So I would go for something like this instead:
```suggestion
      this.router.transitionTo('route-not-found', { wildcard: 'pagina-niet-gevonden' });
```